### ISSUE TITLE
HTML5 not working

### ISSUE THREAD

**Issue by emicom** [YePpHa/YouTubeCenter#1679]
Since the last build 501 the html5 player doesn't play the new videos. It switch to adobe flash. Curiously enough old (some weeks old) videos play fine in html5. Have not explanation, but think that Pale Moon doesn't report correctly it's version to YT.

**Comment by Zarggg** [comment]
Can you provide an example, please?

**Comment by emicom** [comment]
Sure. When I go YT/my subscription - all videos play ONLY on flash. But if I go to some old videos they play html5 without any problem. So is with embed videos - no problem with html5.
I have config YTC to play html5, so I did with YT and requested html5. I've reset YTC to defaults and imported my preferences. After some time when I go and watch some old videos I've watched before they play html5 without problem. Have latest Pale Moon updated. I don't know where lay the problem - YT, PM, or YTC.

**Comment by Zarggg** [comment]
Sorry, let me be more clear: Can you provide some specific videos where the player reverts to Flash when your player is set to HTML5?

I have already checked a few videos released in the past couple days, and all worked with the HTML5 player.

**Comment by emicom** [comment]
https://www.youtube.com/watch?v=kTpGswGmh3g
There is notice on the player for a second saying that my browser is incapable of playing html5.
This video I've watched 2 days ago on flash now play in html5 without problem
https://www.youtube.com/watch?v=r-QFDX7mLqM

**Comment by Zarggg** [comment]
Both of those videos use the HTML5 player in Firefox, so the problem is not likely YTC or YouTube itself.

The best suggestion I can give you is to load http://youtube.com/html5 and make sure your browser supports all relevant technologies. If it does not, your questions would be best directed to the Pale Moon support forum.

**Comment by ErdeFB** [comment]
Last I used Palemoon, it didn't support MSE or at least MSE + H.264. I'd speculate this is somehow related.

**Comment by emicom** [comment]
The HTML5 player is currently used when possible. - is on the html5 page.
I'll try to update the flash - maybe there lay the problem.
Thanks

**Comment by emicom** [comment]
It is something about DASH. On embed videos on pages PM play YT on html5 without problem.
And one more thing - every videos on YT published before april the 1st is playing on htnl5 without problem. What is going on with google?

**Comment by S-ed** [comment]
Youtube featured secure streams (DRM protected), FFox only implemented secure MSE in v38. Flash supports them since april 2014, iirc (since those are Adobe dev).

**Comment by emicom** [comment]
Yes, sure, it is DRM protection. But strangely enough with the new build 503 everything seems fine - html5 is playing good.
